client/src/components/Playground/components/ColabList.js

```diff
@@ -21,6 +21,7 @@ const ColabList = ({colabs}: ColabListProps) => {
           <ColabDetail colab={colab} key={index} />
         ))}
       </ListGroup>
+      <a href="#">Pridaj vlastný report</a>
```

Comment:
There is already a similar button in the banner and its a little bit nicer than this simple link... Maybe you can reuse it if you want...
![image](https://user-images.githubusercontent.com/38027246/54741586-39149e00-4bbf-11e9-93d1-74bf82811397.png)
At least please add some space above the link it doesn't look good...